draft-gpew-priv-ppm.md

```diff
@@ -585,6 +586,13 @@ type 'outdatedConfig'. Clients SHOULD invalidate any cached aggregator
 `HpkeConfig` and retry with a freshly generated Report. If this retried report
 does not succeed, clients MUST abort and discontinue retrying.
 
+The leader MUST ignore any report whose timestamp falls in a batch interval that
+has been aggregated at least once. Otherwise, comparing the aggregate result to
+the previous aggregate result may result in a significant privacy violation.
+
+[NOTE: Differential privacy provides a generic protection against these kinds of
+"batch faults".]
```

Comment:
```suggestion
```

This is not a helpful statement.  It could be misinterpreted as saying that differential privacy is enough.  But anti-replay is critical to ensuring that differential privacy protections are adequate.  This says the exact opposite.

Reply:
Great point, I will just axe this.

The broader point is that we've uncovered a number of potential "batch faults", all of which fall more or less into the umbrella of privacy leaks that DP aims to mitigate. To be clear, I don't think we should *require* DP to be used in the core PPM protocol, but I thought it might be worth noting the connection for later on when we really think through how to incorporate DP.